Essentials/src/main/resources/messages.properties

```diff
@@ -532,6 +532,7 @@ ignoreCommandUsage1Description=Ignores or unignores the given player
 ignoredList=\u00a76Ignored\:\u00a7r {0}
 ignoreExempt=\u00a74You may not ignore that player.
 ignorePlayer=\u00a76You ignore player\u00a7c {0} \u00a76from now on.
+ignoreYourself=\u00a76That won''t solve your problems.
```

Comment:
As much as I appreciate the joke, we might want something a little more descriptive in case someone hits this trying to ignore someone with a similar username/nickname to themselves